pkgs/os-specific/linux/prl-tools/default.nix

```diff
@@ -39,7 +56,8 @@ stdenv.mkDerivation rec {
     fi
   '';
 
-  patches = lib.optionals (lib.versionAtLeast kernel.version "5.18") [ ./prl-tools.patch ];
+  patches = lib.optional (lib.versionAtLeast kernel.version "5.18") ./prl-tools-5.18.patch
+    ++ lib.optional (lib.versionAtLeast kernel.version "5.19") ./prl-tools-5.19.patch;
```

Comment:
`lib.optionals`?

Reply:
`lib.optional` is of type `bool -> a -> [a]`, so it should work. Is `lib.optionals` preferred in nixpkgs?

Reply:
let ask @SuperSandro2000 but I think that yes.

Reply:
Does not really matter as long as we don't end up with nested lists